get-customer-by-org-number/src/test/java/no/unit/nva/customer/get/GetCustomerByOrgNumberHandlerTest.java

```diff
@@ -69,12 +65,14 @@ public void setUp() {
     @SuppressWarnings("unchecked")
     public void getCustomerByOrgNumberReturnsCustomerWhenInputIsExistingCustomerOrgNumber() throws Exception {
         UUID identifier = UUID.randomUUID();
-        Customer customer = new Customer.Builder()
+        CustomerDb customerDb = new CustomerDb.Builder()
             .withIdentifier(identifier)
             .withFeideOrganizationId(SAMPLE_ORG_NUMBER)
             .withCristinId(SAMPLE_CRISTIN_ID)
             .build();
-        when(customerServiceMock.getCustomerByOrgNumber(SAMPLE_ORG_NUMBER)).thenReturn(customer);
+        when(customerServiceMock.getCustomerByOrgNumber(SAMPLE_ORG_NUMBER)).thenReturn(customerDb);
```

Comment:
Could we do this without mocks? Since we have a local dynomodb instance in tests?